content/v2.0/reference/config-options.md

```diff
@@ -174,6 +176,26 @@ influxd --store=bolt
 
 ---
 
+## --tls-cert
+
+Path to TLS certificate file.
+
+```sh
+influxd --tls-cert "influxdb.crt" --tls-key "influxdb.key"
+```
+
+For more information, see [Enable TLS encryption](https://v2.docs.influxdata.com/v2.0/security/enable-tls/).
+
```

Comment:
Add a horizontal rule (`---`) between the two settings to match the formatting of the rest of the doc.